src/include/simeng/CoreInstance.hh

```diff
@@ -14,6 +14,7 @@
 #include "simeng/arch/aarch64/Architecture.hh"
 #include "simeng/arch/aarch64/Instruction.hh"
 #include "simeng/arch/aarch64/MicroDecoder.hh"
+#include "simeng/arch/riscv/Architecture.hh"
```

Comment:
Why is `simeng/arch/riscv/Instruction.hh` not required when the AArch64 version is?

Reply:
Including ```riscv.Architecture.hh``` pulls in the ```Instruction.hh``` header with it. Playing around, it seems that lines 15 and 16 are redundant as including ```aarch64/Architecture.hh``` also pulls in the relevant headers.

Reply:
If the 2 aarch64 headers are not needed, could you remove them please